MediaBrowser.Model/Session/SessionMessageType.cs

```diff
@@ -0,0 +1,50 @@
+#pragma warning disable CS1591
+
+namespace MediaBrowser.Model.Session
+{
+    /// <summary>
+    /// The different kinds of messages that are used in the WebSocket api
```

Comment:
```suggestion
    /// The different kinds of messages that are used in the WebSocket api.
```
For the analyzers ;)